src/board/system76/bonw15/gpio.c

```diff
@@ -65,7 +65,8 @@ void gpio_init() {
     GPDRC = BIT(6);
     // BL_PWM_EN_EC
     GPDRD = BIT(3);
-    GPDRE = 0;
+    // USB_PWR_EN# (inverted)
```

Comment:
Inverted as in it's active high? Should we change it to `USB_PWR_EN` and make a note on the GPIO config below?

Reply:
This is the name in the schematic, so I'm inclined to leave it.